src/models/model_TimeSeriesTableFull.jl

```diff
@@ -2,23 +2,76 @@
 # Do not modify this file directly. Modify the OpenAPI specification instead.
 
 
-@doc raw"""TimeSeriesTableFull
-
-    TimeSeriesTableFull(;
-        name=nothing,
-        description=nothing,
-        geographicIntegration=nothing,
-        sequence=nothing,
-        timeSeries=nothing,
-        geogLevels=nothing,
-    )
-
-    - name::String
-    - description::String
-    - geographicIntegration::String
-    - sequence::Float32
-    - timeSeries::Vector{TimeSeriesTableFullTimeSeriesInner}
-    - geogLevels::Vector{TimeSeriesTableFullTimeSeriesInner}
+"""
+```
+TimeSeriesTableFull(;
+    name=nothing,
+    description=nothing,
+    geographicIntegration=nothing,
+    sequence=nothing,
+    timeSeries=nothing,
+    geogLevels=nothing,
+)
+```
+This function creates a table record for the `TimeSeriesTableFull` with fixed name, description, geographical and timeline information.
```

Comment:
Could you make this description a bit more concise?

Reply:
Should be good now.